browser/ui/webui/settings/brave_wallet_handler.cc

```diff
@@ -356,9 +351,7 @@ void BraveWalletHandler::AddHiddenNetwork(const base::Value::List& args) {
     return;
   }
 
-  PrefService* prefs = GetPrefs();
-  AllowJavascript();
```

Comment:
why is this line removed?

Reply:
Unintentional. Fixed: https://github.com/brave/brave-core/pull/24365/commits/90f3a806f44fb0020b62971c6a305622a0c6a362